tests/test-demo-buildsystem.sh

```diff
@@ -19,9 +19,12 @@
 
 set -euo pipefail
 
-if ! fusermount --version >/dev/null 2>&1; then
+if ! which fusermount >/dev/null 2>&1; then
     echo "1..0 # SKIP no fusermount"
     exit 0
+elif ! capsh --print | grep ^Current | grep -q cap_sys_admin; then
```

Comment:
That makes more sense to me! Any chance someone who sees this failure could commit it, after landing #232? (I don't currently have any container environments that would see this, only VMs, chroots and the real system)